addons/spreadsheet/static/src/o_spreadsheet/migration.js

```diff
@@ -16,6 +16,12 @@ const MAP = {
     "LIST.HEADER": "ODOO.LIST.HEADER",
 };
 
+const MAP_V10 = {
```

Comment:
```suggestion
const MAP_FN_NAMES_V10 = {
```
or something like that, to be more explicit than just `MAP` ?